File: 프로그래머스/3/42884. 단속카메라/단속카메라.js
function solution(routes) {
    var answer = 0;
    
    let dist = -30001
    
    routes = routes.sort((a, b) => (a[0] - b[0], a[1] - b[1]))
    

    for (const route of routes){
        if(route[0] <= dist){
            continue
        }else{
            answer += 1
            dist = route[1]
        }
    }
    
    
    
    return answer;
}

File: 프로그래머스/3/42884. 단속카메라/단속카메라.py
def solution(routes):
    answer = 0
    
    dist = -30001
    routes.sort(key= lambda x: x[1])
    
    for route in routes:
        if route[0] <= dist:
            continue
        else:
            answer += 1
            dist = route[1]
            
    return answer